src/store/index.ts

```diff
@@ -61,6 +62,7 @@ const reducers = combineReducers({
   [COOKIES_REDUCER_ID]: cookies,
   [ADDRESS_BOOK_REDUCER_ID]: addressBook,
   [CURRENT_SESSION_REDUCER_ID]: currentSession,
+  [TRANSACTIONS_NEW]: allTransactions,
```

Comment:
`[TRANSACTIONS]` is free, perhaps it's a good idea to use it.
`[TRANSACTIONS]: allTransactions`.

Reply:
good idea !

Reply:
done